daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -468,6 +485,14 @@ enum BuiltinFunction {
   APPEND_TEXT = 24;
 
   ERROR = 25;
+  THROW = 137;                      // *Available in versions >= 1.dev*
+  ANY_EXCEPTION_MESSAGE = 138;      // *Available in versions >= 1.dev*
+  GENERAL_ERROR_MAKE = 139;         // *Available in versions >= 1.dev*
```

Comment:
How do you feel about moving the `MAKE` to the front?

Reply:
Actually, do we need to expose them in DAML-LF at all? I think we only need those as values in order to explain the exception semantics but we don't need the expression form of this (outside of Speedy).

Reply:
I moved the `MAKE` up front.